xdsl/dialects/utils.py

```diff
@@ -388,3 +392,30 @@ def parse_dynamic_index_list_without_types(
             values.append(value_or_index)
 
     return values, indices
+
+
+# Fast Math Flags
```

Comment:
Please remove

Reply:
How do you feel about a region for them instead of a stray comment?